source/_assets/css/clipboard.css

```diff
@@ -1,5 +1,5 @@
 .code-block-wrapper .copyBtn {
-    @apply absolute right-2 top-2 outline-none text-gray-200 opacity-25;
+    @apply absolute right-2 top-4 outline-none text-gray-200 opacity-25;
```

Comment:
Make it 5 or 6, so that the middle of the button is aligned with the text

Reply:
5 looks perfect. 
<img width="697" alt="image" src="https://user-images.githubusercontent.com/54532330/181447965-6195a4e9-bc47-4b5b-bb71-e1ae51042ade.png">

Reply:
Hmm, 5 seems *slightly* bit too low, but 4 was definitely too high, so I think we'll just go with 5 👍🏻